### ISSUE TITLE
Fine tuning with Resnet50 always gives an accuracy of 50%, what am I doing wrong? 

### ISSUE THREAD

**Issue by Euphemiasama** [fchollet/deep-learning-models#23]
I am using the `keras.applications.resnet50.ResNet50(include_top=False,weights='imagenet')` to load the model without the top fully connected layers, so It downloads the weights [https://github.com/fchollet/deep-learning-models/releases/download/v0.2/resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5] 
since I am using a theano backend. When using Resnet50 model I always obtain an accuracy of 50% while all the other models ; VGG16 , VGG19, Inception V3 give a quite good accuracy ~ 80 %.
What am I doing wrong, can somebody help ?
